### ISSUE TITLE
Inconsistent ordering for group variables of different classes

### ISSUE THREAD

**Issue by maraab23** [maraab23/ggseqplot#5]
Bug reported by Gilbert Ritschard

When the group vector is numeric, the order of the groups is determined by the **order of occurrences** of the group values in the dataset, whereas it is the **order of the levels** when the group vector is a factor.  Example:
  

``` r
library(TraMineR)
library(ggseqplot)

dat <- read.table(text="a a b 3
                  a b b 1
                  b b b 2")
class(dat[,4])
#> [1] "integer"
```

``` r

s <- seqdef(dat[,1:3])

seqiplot(s, group=dat[,4])
```

![](https://i.imgur.com/jExV0IF.png)

``` r
ggseqiplot(s, group=dat[,4])
```

![](https://i.imgur.com/9TpYVeM.png)

``` r
ggseqiplot(s, group=factor(dat[,4]))
```

![](https://i.imgur.com/wwwFRbz.png)

**Possible solution:** Transforming numeric group variables into a factor would avoid this unexpected behavior.

**Comment by maraab23** [comment]
fixed with [43c8c3e](https://github.com/maraab23/ggseqplot/commit/43c8c3ea528f9a326423fc7828346da15af3c22f)

**Comment by maraab23** [comment]
fixed with [43c8c3e](https://github.com/maraab23/ggseqplot/commit/43c8c3ea528f9a326423fc7828346da15af3c22f)
